build/configs/qemu/HowToRunNetworkStackOnQemu.md

```diff
@@ -0,0 +1,91 @@
+# Steps to verify network stack in QEMU is as follows
+1. cd os/tools
+2. ./configure.sh qemu/tc_16m
+3. cd ..
+4. make
+
+Connect WiFi AP to Linux machine(wired connection. dhcp should be enabled in the AP)
+Create bridge and tap interface by adding below commands into shell script file(net_up.sh)
+
+## Bringup interface
+script to bring up the tun device in QEMU in bridged mode first parameter is name of tap device (e.g. tap0)
+```
+#!/bin/sh
+ETH0IPADDR=192.168.1.20
+MASK=255.255.255.0
+GATEWAY=192.168.1.1
+BROADCAST=192.168.1.255
+```
+
+First take eth0 down, then bring it up with IP address 0.0.0.0
```

Comment:
You can # these sentences, and make them appear as BASH comments. Just put all the script body within one ```(start) and ```(end)